pdns/recursordist/test-mtasker.cc

```diff
@@ -40,6 +41,43 @@ BOOST_AUTO_TEST_CASE(test_Simple)
   BOOST_CHECK_EQUAL(g_result, o);
 }
 
+static const size_t stackSize = 8 * 1024;
+static const size_t headroom = 256; // Decrease to hit stackoverflow
```

Comment:
Right the required headrom is a lot lower for me (112), but I don't quite see how to determine that automatically without too much trouble.

Reply:
128 did not work on arm64, so i picked the next power of two